### PR TITLE
#2221 refactor qstatus at init v2

### PR DESCRIPTION
**Pull request by marcochiaramello**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non regression tests were added (new model, new feature and bug fix)
- [ ] main documentation was updated (update of input/output file, 3rd party, model, repository organization, solver)
- [ ] example documentations were updated (new example in examples folder)
- [ ] if this PR modifies the parameters or inputs/outputs of a model/solver: the corresponding xsl was added in util/xsl and platform DB were updated
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated
- [ ] if this commit targets a release branch: the corresponding milestone was added in the ticket and in this PR

### CODE REVIEW

**Review by cossartque — CHANGES_REQUESTED** [review]
It seems good to me, thanks.
Just one remark in several models, there is no line before import and two spaces before annotation.

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPVRemote_INIT.mo [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  extends BaseClasses_INIT.BaseGeneratorSignalN_INIT;
+  extends AdditionalIcons.Init;
+
+  type QStatus = enumeration (Standard "Reactive power is fixed to its initial value",
+                              AbsorptionMax "Reactive power is fixed to its absorption limit",
+                              GenerationMax "Reactive power is fixed to its generation limit");
+
+  parameter Types.VoltageModule URef0 "Start value of the voltage regulation set point in kV";
+  parameter Types.VoltageModule URegulated0 "Start value of the regulated voltage in kV";
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPVRemote_INIT.mo:28]: Why do you need it ?

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPVRemoteSFR_INIT.mo [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  extends BaseClasses_INIT.BaseGeneratorSignalN_INIT;
+  extends AdditionalIcons.Init;
+
+  type QStatus = enumeration (Standard "Reactive power is fixed to its initial value",
+                              AbsorptionMax "Reactive power is fixed to its absorption limit",
+                              GenerationMax "Reactive power is fixed to its generation limit");
+
+  parameter Types.VoltageModule URef0 "Start value of the voltage regulation set point in kV";
+  parameter Types.VoltageModule URegulated0 "Start value of the regulated voltage in kV";
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPVRemoteSFR_INIT.mo:28]: Why do you need it ?

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPVRemote_INIT.mo (hunk 1 above)
Comment (marcochiaramello) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPVRemote_INIT.mo:28]: it is used to determine the qstatus initial value (to ahve the comparison). In the model itself it is not useful as there is a connection to the monitored bus. And yes, an update on dfl + interface is need, it is ongoing

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPVDiagramPQ.mo [hunk 3]
```diff
@@ -28,12 +28,15 @@ model GeneratorPVDiagramPQ "Model for generator PV based on SignalN for the freq
   parameter String QMinTableFile "Text file that contains the table to get QMinPu from PGenPu";
   parameter String QMaxTableFile "Text file that contains the table to get QMaxPu from PGenPu";
   parameter Types.ReactivePower QNomAlt "Nominal reactive power of the generator on alternator side in Mvar";
+  parameter Boolean limUQUp0 "Whether the maximum reactive power limits are reached or not (from generator voltage regulator), start value";
```
[the remark this answers is not in the document]
Comment (marcochiaramello) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPVDiagramPQ.mo:31]: done

**Review by cossartque — APPROVED** [review]

**Review by cossartque — APPROVED** [review]

**Review by cossartque — APPROVED** [review]

**Review by cossartque — APPROVED** [review]

**Pull request merged**